docs/Contributing/Testing-and-local-development.md

```diff
@@ -574,3 +588,5 @@ Reference the [Apple DEP Profile documentation](https://developer.apple.com/docu
 2. In ABM, look for the computer with the serial number that matches the one your VM has, click on it and click on "Edit MDM Server" to assign that computer to your MDM server.
 
 3. Boot the machine, it should automatically enroll into MDM.
+
+<meta name="pageOrderInSection" value="1500">
```

Comment:
?

Reply:
This is needed for publishing the markdown files on the website.
https://github.com/fleetdm/fleet/blob/0fa82fa8a590f4054a2cc5059f594b656b188d30/handbook/engineering/README.md#page-order